core/src/main/scala/cats/instances/map.scala

```diff
@@ -79,4 +80,23 @@ trait MapInstances extends cats.kernel.instances.MapInstances {
 
     }
   // scalastyle:on method.length
+
+}
+
+trait MapInstancesBinCompat0 {
+
+  implicit def catsStdComposeForMap: Compose[Map] = new Compose[Map] {
```

Comment:
Should this be a `val` instead of a `def` so the same instance can be reused repeatedly?